pkg/api/api.go

```diff
@@ -247,9 +247,9 @@ func (hs *HTTPServer) registerRoutes() {
 
 		// auth api keys
 		apiRoute.Group("/auth/keys", func(keysRoute routing.RouteRegister) {
-			keysRoute.Get("/", routing.Wrap(GetAPIKeys))
-			keysRoute.Post("/", quota("api_key"), bind(models.AddApiKeyCommand{}), routing.Wrap(hs.AddAPIKey))
-			keysRoute.Delete("/:id", routing.Wrap(DeleteAPIKey))
+			keysRoute.Get("/", authorize(reqOrgAdmin, accesscontrol.ActionApiKeyList, accesscontrol.ScopeUsersAll), routing.Wrap(GetAPIKeys))
+			keysRoute.Post("/", authorize(reqOrgAdmin, accesscontrol.ActionApiKeyAdd, accesscontrol.ScopeUsersAll), quota("api_key"), bind(models.AddApiKeyCommand{}), routing.Wrap(hs.AddAPIKey))
```

Comment:
```suggestion
			keysRoute.Post("/", authorize(reqOrgAdmin, accesscontrol.ActionApiKeyAdd), quota("api_key"), bind(models.AddApiKeyCommand{}), routing.Wrap(hs.AddAPIKey))
```

Reply:
I dont thing you need to have scope for this

Reply:
So no scope is all scope?